flow_control/flow.py

```diff
@@ -10,7 +10,7 @@
 from xblock.fields import Scope, Integer, String
 from xblockutils.studio_editable import StudioEditableXBlockMixin
 from xblock.validation import ValidationMessage
-from courseware.model_data import ScoresClient
+from lms.djangoapps.courseware.model_data import ScoresClient
```

Comment:
You beat me to this change! We'll merge your PR but on this line we'll add a small conditional import so our tests work correctly.

Thank you for your contribution!